docs/preview/03-Features/powershell/azure-logic-apps-consumption.md

```diff
@@ -149,13 +149,13 @@ Typically done the first task of the release pipeline, right before the deployme
 The Azure Logic Apps to be disabled and the order in which this will be done, will be defined in the provided configuration file.
 The order of the Azure Logic Apps in the configuration file (bottom to top) defines the order in which they will be disabled by the script. The counterpart of this script used to enable the Azure Logic Apps, will take the order as specified (top to bottom) in the file.
 
-| Parameter         | Mandatory | Description                                                                                                                                                 |
-| ----------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| ResourceGroupName | yes       | The resource group containing the Azure Logic Apps.                                                                                                         |
-| DeployFileName    | no        | If your solution consists of multiple interfaces, you can specify the flow-specific name of the configuration file.                                         |
-| ResourcePrefix    | no        | In case the Azure Logic Apps all start with the same prefix, you can specify this prefix through this parameter instead of updating the configuration-file. | 
-| EnvironmentName   | no        | The name of the Azure environment where the Azure Logic App resides. (default: `AzureCloud`)                                                                |
-| ApiVersion        | no        | The version of the management API to be used.  (default: `2016-06-01`)                                                                                      |
+| Parameter           | Mandatory | Description                                                                                                                                                 |
+| ------------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `ResourceGroupName` | yes       | The resource group containing the Azure Logic Apps.                                                                                                         |
+| `DeployFileName`    | yes       | If your solution consists of multiple interfaces, you can specify the flow-specific name of the configuration file.                                         |
```

Comment:
Was this `DeployFileName` set to mandatory something wrong with the docs, or is this a breaking change?

Reply:
Something wrong with the docs, the `DeployFileName` should be mandatory 😄